lrrbot/__init__.py

```diff
@@ -172,7 +170,9 @@ def wrapper(function):
 	def on_connect(self, conn, event):
 		"""On connecting to the server, join our target channel"""
 		log.info("Connected to server")
-		self.twitchclient(conn, 3)
+		conn.cap("REQ", "twitch.tv/membership") # get join/part messages
```

Comment:
The only thing we use `JOIN`/`PART` for is checking for `:lrrbot JOIN #loadingreadyrun` and that is sent even without this capability.